packages/button/index.js

```diff
@@ -34,10 +35,18 @@ class WarpButton extends kebabCaseAttributes(WarpElement) {
     rel: { type: String, reflect: true },
     fullWidth: { type: Boolean, reflect: true },
     buttonClass: { type: String, reflect: true },
+    name: { type: String, reflect: true },
+    value: { type: String, reflect: true },
```

Comment:
Every element that is to be associated with a form needs to support name and value